.github/workflows/ci-pipeline.yml

```
@@ -68,3 +68,10 @@ jobs:
           asset_path: ${{ steps.release.outputs.artifacts_archive_path }}
           asset_name: ${{ steps.release.outputs.artifacts_archive_path }}
           asset_content_type: application/zip
+      - if: github.event.release
+        name: Build & Push Docker images
+        id: docker
+        timeout-minutes: 10
+        run: |
+          echo ${{ secrets.GHCR_TOKEN }} | docker login ghcr.io -u $GITHUB_ACTOR --password-stdin
```

Lets changes that to github token, seem to be a better practice
```suggestion
          echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u $GITHUB_ACTOR --password-stdin
```

 * https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry
 * https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow